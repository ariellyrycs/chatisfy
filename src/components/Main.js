import React, { useState } from 'react';
import ChatDrawer from '../components/ChatDrawer';
import Enter from '../components/Enter';

import { Container } from '@material-ui/core';


function Main() {
    const [isLoged, setLog] = useState(false);
    return (<Container fixed>
        {isLoged ?
            <ChatDrawer />:
            <Enter setLog={setLog} />}
    </Container>
    )
}
export default Main;