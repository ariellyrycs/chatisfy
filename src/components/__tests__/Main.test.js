import React from 'react';
import Main from '../Main';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json'

describe('<Main />', () => {

    test('should render main component', () => {
        const tree = shallow(<Main/>);
        expect(toJson(tree)).toMatchSnapshot();
    });
});