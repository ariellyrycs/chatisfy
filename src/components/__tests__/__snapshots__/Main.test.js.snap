// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> should render main component 1`] = `
<WithStyles(ForwardRef(Container))
  fixed={true}
>
  <Enter
    setLog={[Function]}
  />
</WithStyles(ForwardRef(Container))>
`;
